R/runEdgeRNorm.R

```diff
@@ -48,7 +48,8 @@ runEdgeRNorm <- function(dgeObj,
                               itemType = "DGEList",
                               funArgs = funArgs,
                               itemAttr = itemAttr,
-                              parent = "counts")
+                              parent = "counts",
+                              overwrite = TRUE)
```

Comment:
I am not sure if it's the correct fix; overwrite = TRUE . Please confirm this with Connie and also check if it's working without overwrite = T with GECO object.

Reply:
As far as I know, the overwrite here is not saved in the RDS and returned from the function which makes sense as the function updates the passed object. Thanks